### PR TITLE
Update bg.m3u

### PR DESCRIPTION
**Pull request by SileC**
New channel

### CODE REVIEW

File: channels/bg.m3u [hunk 1]
```diff
@@ -5,6 +5,8 @@ http://100automoto.tv:1935/bgtv1/autotv/playlist.m3u8
 http://alfa.bg:1935/live/live/playlist.m3u8
 #EXTINF:-1 tvg-id="Alfa TV BG" tvg-name="Alfa TV BG" tvg-language="Bulgarian" tvg-logo="https://i.imgur.com/UaXwtnI.png" group-title="News",Alfa TV
 http://alfa.bg:1935/live/live/playlist.m3u8?fluxustv.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Bulgarian" tvg-logo="https://www.dafont.com/forum/attach/orig/7/4/747968.png" group-title="Music",Balkanica
+https://iptv-org.github.io/iptv/index.m3u
```
Comment (freearhey) [inline channels/bg.m3u]: The link is not correct.

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by SileC** [comment]
this is a rtsp not m3u link

**Comment by freearhey** [comment]
no, this link just leads to our playlist

**Comment by SileC** [comment]
I saw that, that is why I closed this pull request and I send another one with correct link

**Pull request closed without merging**